zerver/tests/test_mattermost_importer.py

```diff
@@ -16,51 +16,67 @@
 from zerver.data_import.mattermost import mattermost_data_file_to_dict, process_user, convert_user_data, \
     create_username_to_user_mapping, label_mirror_dummy_users, reset_mirror_dummy_users, \
     convert_channel_data, write_emoticon_data, get_mentioned_user_ids, check_user_in_team, \
-    build_reactions, get_name_to_codepoint_dict, do_convert_data
+    build_reactions, get_name_to_codepoint_dict, do_convert_data, convert_huddle_data, \
+    generate_huddle_name
 from zerver.data_import.sequencer import IdMapper
 from zerver.data_import.import_util import SubscriberHandler
-from zerver.models import Reaction, UserProfile, Message, get_realm
+from zerver.models import Reaction, UserProfile, Message, get_realm, Recipient
 
 class MatterMostImporter(ZulipTestCase):
     logger = logging.getLogger()
     # set logger to a higher level to suppress 'logger.INFO' outputs
     logger.setLevel(logging.WARNING)
 
-    def setUp(self) -> None:
+    def test_mattermost_data_file_to_dict(self) -> None:
         fixture_file_name = self.fixture_file_name("export.json", "mattermost_fixtures")
-        self.mattermost_data = mattermost_data_file_to_dict(fixture_file_name)
-        self.username_to_user = create_username_to_user_mapping(self.mattermost_data["user"])
```

Comment:
Does it make sense to do the migration of these things out of the `setUp` function to a separate commit?  Currently the test changes are really hard to read, and I think this would address a lot of that.

Reply:
Done.